controllers/v1/solutions.js

```diff
@@ -16,64 +16,221 @@ module.exports = class  Solutions extends Abstract{
     return "solutions";
   }
 
-      /**
-    * @api {post} /improvement-project/api/v1/solutions/create Create solution
-    * @apiVersion 1.0.0
-    * @apiName Create solution
-    * @apiGroup Solutions
-    * @apiParamExample {json} Request-Body:
-    * {
-    "programExternalId" : "AMAN_TEST_123-1607937244986",
-    "entityType" : "school",
-    "externalId" : "IMPROVEMENT-PROJECT-TEST-SOLUTION",
-    "name" : "Improvement project test solution",
-    "description" : "Improvement project test solution"
-    }
-    * @apiHeader {String} internal-access-token internal access token  
-    * @apiHeader {String} X-authenticated-user-token Authenticity token
-    * @apiSampleRequest /kendra/api/v1/solutions/create
-    * @apiUse successBody
-    * @apiUse errorBody
-    * @apiParamExample {json} Response:
-    * {
-    "message": "Improvement project solution created successfully",
-    "status": 200,
-    "result": {
-        "_id": "6006a94d67f675771573226d"
-    }
-  }
-    */
+  /**
+* @api {post} /improvement-project/api/v1/solutions/create Create solution
+* @apiVersion 1.0.0
+* @apiName Create solution
+* @apiGroup Solutions
+* @apiParamExample {json} Request-Body:
+* {
+"programExternalId" : "AMAN_TEST_123-1607937244986",
+"entityType" : "school",
+"externalId" : "IMPROVEMENT-PROJECT-TEST-SOLUTION",
+"name" : "Improvement project test solution",
+"description" : "Improvement project test solution"
+}
+* @apiHeader {String} internal-access-token internal access token  
+* @apiHeader {String} X-authenticated-user-token Authenticity token
+* @apiSampleRequest /kendra/api/v1/solutions/create
+* @apiUse successBody
+* @apiUse errorBody
+* @apiParamExample {json} Response:
+* {
+"message": "Improvement project solution created successfully",
+"status": 200,
+"result": {
+    "_id": "6006a94d67f675771573226d"
+}
+}
+*/
 
-     /**
-   * Create solution.
-   * @method
-   * @name create
-   * @param {Object} req - requested data.
-   * @param {String} req.body - requested body data.
-   * @returns {JSON} Created solution data.
-   */
+  /**
+ * Create solution.
+ * @method
+ * @name create
+ * @param {Object} req - requested data.
+ * @param {String} 
+ * @returns {JSON} Created solution data.
+ */
 
   async create(req) {
     return new Promise(async (resolve, reject) => {
       try {
 
-        let createdSolution = await solutionsHelper.create(
+        let solutionData = await solutionsHelper.createSolution(
           req.body,
-          req.userDetails.userToken
+          true
         );
 
-        createdSolution["result"] = createdSolution.data;
+        solutionData["result"] = solutionData.data;
```

Comment:
@Prajwal17Tunerlabs can you move this to helper and send the result from helper itself